app/views/devise/sessions/new.html.haml

```diff
@@ -17,3 +17,5 @@
               = f.govuk_check_box :remember_me, true, multiple: false, label: { text: "Remember me" }
       .actions
         = f.govuk_submit "Log in"
+
+    = link_to "Forgot password?", new_user_password_path
```

Comment:
If I do `rails g devise:views`, the `sessions#new` ERB comes with the following line:

`<%= render "devise/shared/links" %>`

I guess we've decided to drop it because this is more flexible when we have `rememberable` [turned on](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/1570/commits/63d5c9db8a83f1b2ba8e43b90f099def3b3ab305#diff-5fe3b11f41a498ce1b5888e455dce9420646cfe49a8e100a39d31d31eb70e041R15)?

Reply:
We decided that the links weren't serving us for now, so the simpler the better.